### ISSUE TITLE
Extensions details view needs accessibility polish

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#88792]
1. Keyboard focus should be in logical order
2. Screen reader does not announces the selected/not selected state "Details, Extension Pack, Contribution changelog" buttons
3. Keyboard focus is not visible on the "scroll up" button

**Comment by isidorn** [comment]
@sandy081 can we open this up for PRs or give it to some intern? Feels like a pretty isolated part of VS Code so I think it is a good candidate.

**Comment by sandy081** [comment]
Sure that would be helpful.

**Comment by isidorn** [comment]
Ok, let me add help wanted / feature request and we will see if some enthusiast appears :)

**Comment by xinyeah** [comment]
I'm going to take a shot at this for Grace Hopper's Open Source Day!

**Comment by isidorn** [comment]
This issue is quite demanding as it requires a big rewrite of the extension view.
However we also accept minor improvements

Though here's a code pointer 
https://github.com/microsoft/vscode/blob/b8e87aae6dca78fe0f8dbffd9d6a34921d9913f4/src/vs/workbench/contrib/extensions/browser/extensionEditor.ts#L166
